### PR TITLE
Include top module file when formatting

### PR DESCRIPTION
**Pull request by bwestlin**
### Checklist

* [x] I have read the [Contributor Guide](../../CONTRIBUTING.md)
* [x] I have read and agree to the [Code of Conduct](../../CODE_OF_CONDUCT.md)
* [x] I have added a description of my changes and why I'd like them included in the section below

### Description of Changes

Also format the top module file that is generated if formatting is enabled

### CODE REVIEW

**Review by MarcusGrass — APPROVED** [review]
Real nice no-disk formatting, LGTM

**Comment by bwestlin** [comment]
> Real nice no-disk formatting, LGTM

Thx! Sorry for not waiting on approval :sweat_smile:

**Pull request merged**
